README.md

```diff
@@ -32,9 +32,15 @@ date, please check the [Gerrit vdsm project](https://gerrit.ovirt.org/q/project:
 
 ## Manual installation
 
-Add ovirt repositories to your repositories list:
+Add ovirt repositories to your repositories list.
 
-    sudo dnf install -y http://resources.ovirt.org/pub/yum-repo/ovirt-release-master.rpm
+For CentOS Stream 8 use:
+
+    dnf copr enable -y ovirt/ovirt-master-snapshot centos-stream-8
+    dnf install -y ovirt-release-master
```

Comment:
Old command used sudo, like rest of the file. I guess dropping sudo was not your intent?